pkg/innerring/processors/neofs/process_assets.go

```diff
@@ -34,12 +34,34 @@ func (np *Processor) processDeposit(deposit *neofsEvent.Deposit) {
 		np.log.Error("can't transfer assets to balance contract", zap.Error(err))
 	}
 
-	// fixme: send gas when ^ tx accepted
-	// fixme: do not send gas to the same user twice per epoch
-	err = np.morphClient.TransferGas(deposit.To(), util.Fixed8FromInt64(2))
+	curEpoch := np.epochState.EpochCounter()
+	receiver := deposit.To()
+
+	// check if receiver already received some mint GAS emissions
+	// we should lock there even though LRU cache is already thread save
+	// we lock there because GAS transfer AND cache update must be atomic
+	np.mintEmitLock.Lock()
+	defer np.mintEmitLock.Unlock()
+
+	val, ok := np.mintEmitCache.Get(receiver.String())
+	if ok && val.(uint64) >= curEpoch-np.mintEmitThreshold {
```

Comment:
Can we use  `val.(uint64)+np.mintEmitThreshold >= curEpoch` expression since overflow is much less likely in it?

Reply:
Sure. Fixed.